### PR TITLE
[CELEBORN-1237][METRICS] Refactor metrics name.

### PR DESCRIPTION
**Pull request by leixm**
### What changes were proposed in this pull request?
Refactor metrics name.

### Why are the changes needed?
Easier to understand the meaning of metrics

### Does this PR introduce _any_ user-facing change?
METRICS.md
migration.md
monitoring.md

### How was this patch tested?
Existing UTs.

### CODE REVIEW

**Comment by leixm** [comment]
cc @AngersZhuuuu

**Comment by AngersZhuuuu** [comment]
Pls crate a jira ticket and write pr title as `[CELEBORN-xxx][XX] title`

File: worker/src/main/scala/org/apache/celeborn/service/deploy/worker/FetchHandler.scala [hunk 1]
```diff
@@ -60,11 +60,11 @@ class FetchHandler(
   var registered: AtomicBoolean = new AtomicBoolean(false)
 
   def init(worker: Worker): Unit = {
-    workerSource.addGauge(WorkerSource.CHUNK_STREAM_COUNT) { () =>
+    workerSource.addGauge(WorkerSource.ACTIVE_CHUNK_STREAM_COUNT) { () =>
       chunkStreamManager.getStreamsCount
     }
 
-    workerSource.addGauge(WorkerSource.CREDIT_STREAM_COUNT) { () =>
+    workerSource.addGauge(WorkerSource.ACTIVE_CREDIT_STREAM_COUNT) { () =>
```
Comment (AngersZhuuuu) [inline worker/src/main/scala/org/apache/celeborn/service/deploy/worker/FetchHandler.scala:67]: Also pls update doc

File: worker/src/main/scala/org/apache/celeborn/service/deploy/worker/FetchHandler.scala [hunk 2]
```diff
@@ -60,11 +60,11 @@ class FetchHandler(
   var registered: AtomicBoolean = new AtomicBoolean(false)
 
   def init(worker: Worker): Unit = {
-    workerSource.addGauge(WorkerSource.CHUNK_STREAM_COUNT) { () =>
+    workerSource.addGauge(WorkerSource.ACTIVE_CHUNK_STREAM_COUNT) { () =>
```
Comment (AngersZhuuuu) [inline worker/src/main/scala/org/apache/celeborn/service/deploy/worker/FetchHandler.scala:63]: ditto

**Comment by AngersZhuuuu** [comment]
cc @waitinfuture @SteNicholas @FMX

**Comment by leixm** [comment]
Done.

File: METRICS.md [hunk 3]
```diff
@@ -92,7 +92,8 @@ Here is an example of Grafana dashboard importing.
 |             OfferSlotsTime             |      master       |                                             The time of offer slots.                                              |
 |             OpenStreamTime             |      worker       |             OpenStream means read a shuffle file and send client about chunks size and stream index.              |
 |             FetchChunkTime             |      worker       |                       FetchChunk means read a chunk from a shuffle file and send to client.                       |
-|            ChunkStreamCount            |      worker       |                     The stream count for reduce partition reading streams in current worker.                      |
+|         ActiveChunkStreamCount         |      worker       |                 The active stream count for reduce partition reading streams in current worker.                   |
+|         ActiveCreditStreamCount        |      worker       |                 The active stream count for map partition reading streams in current worker.                      |
```
Comment (AngersZhuuuu) [inline METRICS.md:96]: Remove origin CreditStreamCount

**Comment by leixm** [comment]
Done.

**Review by AngersZhuuuu — APPROVED** [review]
LGTM

File: docs/migration.md [hunk 4]
```diff
@@ -21,6 +21,11 @@ license: |
 
 # Migration Guide
 
+## Upgrading from 0.4 to 0.5
+- Since 0.5.0 Celeborn worker metrics `ChunkStreamCount` is renamed as `ActiveChunkStreamCount`.
```
Comment (pan3793) [inline docs/migration.md:25]: nit: add a blank line

**Review by pan3793 — APPROVED** [review]
LGTM

File: docs/migration.md (hunk 4 above)
Comment (leixm) [reply docs/migration.md:26]: done.

**Pull request closed without merging**